Jenkinsfile

```diff
@@ -388,8 +388,18 @@ pipeline {
         always {
           junit testResults: '**/target/surefire-reports/*.xml'
           container('maven') {
-            // clean up the redis namespace
-            sh "kubectl delete namespace ${NAMESPACE_REDIS} --ignore-not-found=true"
+            script {
+              try {
+                 // clean up the redis namespace
```

Comment:
Can you please indent with 2 spaces instead of 3

Reply:
done